src/js/auth.js

```diff
@@ -14,49 +14,55 @@
  * limitations under the License.
  */
 
-/* global gapi */
-
 // Utility methods for dealing with Google auth:
-// https://developers.google.com/identity/sign-in/web/reference
+// https://developers.google.com/identity/oauth2/web/guides/use-token-model
 
-export async function checkAuthStatus() {
-  // This is set on the window in `index.html`.
-  await window.authorizeUser;
+window.tokenPromise = new Promise((resolve, reject) => {
+  window.tokenLoadOkay = resolve;
+  window.tokenLoadFail = reject;
+});
+
+let tokenClient;
+let accessToken;
 
-  return getAuthInstance().isSignedIn.get();
+const clientId = window.clientId;
```

Comment:
move these two lines inside of initAuthClient?  or .. just reference the globals directly on L33 and L34

Reply:
Done.